src/Kestrel.Https/KestrelServerOptionsHttpsExtensions.cs

```diff
@@ -0,0 +1,25 @@
+﻿// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using Microsoft.AspNetCore.Server.Kestrel.Core;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.Https
+{
+    public static class KestrelServerOptionsHttpsExtensions
+    {
+        public static void ConfigureHttpsDefaults(this KestrelServerOptions serverOptions, Action<HttpsConnectionAdapterOptions> configureOptions)
```

Comment:
I would add doc comments to this and `ConfigureEndpointDefaults` need to be called prior to the Configure/Listen calls it is providing default values for.